crates/compiler/test_gen/Simp.roc

```diff
@@ -0,0 +1,253 @@
+# an extremely simple encoder/decoder formatter
+# module [
+#     Simp,
+#     simp,
+# ]
```

Comment:
Clever solution. It might be helpful to also leave a comment here explaining why this isn't a real module -- we inline it as a string into the tests.